federated-css/e2e/checkReactTailwindModuleAndJss.cy.ts

```diff
@@ -0,0 +1,41 @@
+import { BaseMethods } from '../../cypress/common/base';
+import {
+    selectors,
+} from '../../cypress/common/selectors';
+import { CssAttr } from '../../cypress/types/cssAttr';
+import { ExposedButtons } from './constants';
+
+const basePage: BaseMethods = new BaseMethods();
+const pagePort = 3007;
+
+describe("Check React Tailwind Module and Jss", () => {
+    it('should verify consumed jss page', () => {
+      basePage.openLocalhost(pagePort, `#/${ExposedButtons.jss.path}`);
+      basePage.checkElementHaveProperty({
+        selector: selectors.federatedCssButton,
+        prop: CssAttr.backgroundColor,
+        value: ExposedButtons.jss.bgColor,
+      });
+    });
+
+    it('should verify consumed tailwind module page', () => {
+      basePage.openLocalhost(pagePort, `#/${ExposedButtons.tailwindBlue.path}`);
+      basePage.checkElementHaveProperty({
+        selector: selectors.federatedCssButton,
+        prop: CssAttr.backgroundColor,
+        value: ExposedButtons.tailwindBlue.bgColor,
+      });
+    });
+
+    it('should verify exposed buttons colors', () => {
+      basePage.openLocalhost(pagePort, '#/combined');
+      [ExposedButtons.jss, ExposedButtons.tailwindBlue].forEach((cfg, idx) => {
+        basePage.checkElementHaveProperty({
+          selector: selectors.federatedCssButton,
+          prop: CssAttr.backgroundColor,
+          value: cfg.bgColor,
+          index: idx,
+        });
+      });
+    });
```

Comment:
as I understand you have almost similar checks which differs only by port. If I right please open any commonChecks.cy.ts file and check how we can handle common tests with usage of forEach. It's very likely that your tests would become much smaller with this approach